src/Symfony/Bridge/Doctrine/Tests/DataCollector/DoctrineDataCollectorTest.php

```diff
@@ -74,7 +75,7 @@ public function testCollectTime()
     /**
      * @dataProvider paramProvider
      */
-    public function testCollectQueries($param, $types, $expected, $explainable)
+    public function testCollectQueries($param, $types, $expected, $explainable, bool $runnable)
```

Comment:
can we make it default to `true` and save patching the lines that don't need it in the provider?